wtforms_appengine/__init__.py

```diff
@@ -1 +1,5 @@
 __version__ = '0.1.1dev'
+
+from .ndb import model_form, ModelConverter
```

Comment:
It looks like these lines were added in by @xlevus so you'd have to ask him. I just merged his entire branch into this PR and this was part of it. I assumed it helped with the functionality of his work.